build.ps1

```diff
@@ -234,6 +234,13 @@ $SDKArchs = @($SDKs | ForEach-Object {
 })
 
 # Build functions
+function Invoke-BuildStep([string]$Name) {
+  & $Name @Args
+  if ($Name.Replace("Build-", "") -eq $BuildTo) {
+    exit 0
+  }
+}
```

Comment:
This feels like it shouldn't be part of this change.

Reply:
Oopsie! You didn't see this